test/nested/apps_1177_native_indirect_override.wdl

```diff
@@ -0,0 +1,114 @@
+version 1.1
+
+workflow apps_1177_native_indirect_override {
+
+  call apps_1177_default_instance {
+    input:
+      mock_input = "apps_1177_default_isntance"
```

Comment:
Minor: typo isntance

Reply:
fixed in [6e71262](https://github.com/dnanexus/dxCompiler/pull/339/commits/6e712625cf217365e57d565b513ed83d6731457f)